Story/RainMeadow.StoryHooks.cs

```diff
@@ -327,14 +327,21 @@ private void RainWorldGame_GoToDeathScreen(On.RainWorldGame.orig_GoToDeathScreen
         {
             if (OnlineManager.lobby != null && OnlineManager.lobby.gameMode is StoryGameMode)
             {
-                if (!OnlineManager.lobby.isOwner)
-                {
-                    OnlineManager.lobby.owner.InvokeRPC(RPCs.MovePlayersToDeathScreen);
-                }
-                else
+                foreach (OnlinePlayer player in OnlineManager.players)
```

Comment:
This changes it from host authoritative to any client has the power to tell other games to go to deathscreen